libs/wire-api/src/Wire/API/Message.hs

```diff
@@ -206,6 +210,78 @@ protolensToQualifiedNewOtrMessage protoMsg = do
 protolensToClientId :: Proto.Otr.ClientId -> ClientId
 protolensToClientId = newClientId . view Proto.Otr.client
 
+--- functions to generate protobuf messages
+
+mkQualifiedOtrPayload :: ClientId -> [(Qualified UserId, ClientId, ByteString)] -> ByteString -> ClientMismatchStrategy -> Proto.Otr.QualifiedNewOtrMessage
+mkQualifiedOtrPayload sender recipients dat strat =
```

Comment:
Could these functions here have a unit round trip test? Given a few haskell values, make a proto OTR payload, then decode them again and expect the same?
(I'm fine to add that test in a follow-up PR also; mostly to guard against future regressions)

Reply:
Update: Ah I see these functions were moved from the tests into wire-api. Still, some roundtrip tests might be nice as a FUTUREWORK.

Reply:
Yes, definitely. I added a FUTUREWORK.